src/Sylius/Component/Order/Model/Adjustment.php

```diff
@@ -94,12 +94,7 @@ public function getAdjustable(): ?AdjustableInterface
         if (null !== $this->orderItem) {
             return $this->orderItem;
         }
-
-        if (null !== $this->orderItemUnit) {
-            return $this->orderItemUnit;
-        }
-
-        return null;
+        return $this->orderItemUnit;
```

Comment:
We could go with `return $this->order ?? $this->orderItem ?? $this->orderItemUnit;` here

Reply:
That would suffer the same cognitive overload as https://github.com/Sylius/Sylius/pull/9902/files#r231438056

Code should be as short and as readable as possible.

Reply:
This is my thought process when reading this code:

1. Return order if it exists
2. Return order item if it exists
3. Return order item unit 

When it previously was:

1. Return order if it exists
2. Return order item if it exists
3. Return order item unit if it exists
4. Return null

This change makes me think whether `orderItemUnit` is always set or not (so that I need to go up and look it up) and also makes the diff with possible changes bigger (since if you want to add some logic after order item unit is also null, you have to remove than return, switch it into an if and add your logic).

Reply:
As for `??` usage, I think both `$a ?? $b` and `$a ?: $b` are quite idiomatic in PHP.

Reply:
> We could go with return $this->order ?? $this->orderItem ?? $this->orderItemUnit; here

I get it now :+1: I'll try to implement this.